system/include/libcxx/__config

```diff
@@ -39,6 +39,11 @@
 #define _LIBCPP_ABI_VERSION 1
 #endif
 
+#ifdef defined(__EMSCRIPTEN__) // XXX EMSCRIPTEN: Added to avoid making symbols as visibility(__default__) which for the wasm backend will export symbols to JS
+// FIXME(https://github.com/kripken/emscripten/issues/7383)
+#define _LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS
```

Comment:
Minor detail on how to implement this: Another option is to add this define in `tools/system_libs.py`. In general I think fewer changes to libc++ code is better, so I'd lean towards that, but maybe I'm missing something?

Reply:
I agree with @kripken here.